admin/load.php

```diff
@@ -406,6 +406,40 @@ function perflab_admin_pointer( $hook_suffix ) {
 	$current_user = get_current_user_id();
 	$dismissed    = explode( ',', (string) get_user_meta( $current_user, 'dismissed_wp_pointers', true ) );
 
+	/*
+	 * Show an admin pointer if module is active to prompt
+	 * for an action to use the standalone plugin instead.
+	 */
+	$available_module_names = perflab_get_modules_compatible_with_standalone_plugin();
+	if (
+		! empty( $available_module_names )
```

Comment:
If we change the function, then this can become something like the following which is less confusing:
```suggestion
	/*
	 * If there are any active modules with inactive standalone plugins,
	 * show an admin pointer to prompt the user to migrate.
	 */
	$active_modules_with_inactive_plugins = perflab_get_active_module_data_with_inactive_standalone_plugins();
	if (
		! empty( $active_modules_with_inactive_plugins )
```